docs/_posts/examples/3400-01-26-game-controls.html

```diff
@@ -0,0 +1,133 @@
+---
+layout: example
+category: example
+title: Game-like controls
+description: Move around the map with game-like controls.
+---
+
+<link href='https://www.mapbox.com/base/latest/base.css' rel='stylesheet' />
+<style>
+    /* Dark attribution */
+    .mapboxgl-ctrl.mapboxgl-ctrl-attrib { background-color:rgba(0,0,0,0.5); }
+    .mapboxgl-ctrl.mapboxgl-ctrl-attrib a { color:#fff; }  
+
+    .button-container {
+        height: 100px;
+        width: 130px;
+        position: absolute;
+        bottom: 0;
+        right: 0px;
+        padding: 0;
+        margin: 20px;
+    }
+
+    button {
+        margin: 2.5px;
+    }
+
+    button.round-top {
+        margin: 4px 44px;
+    }
+
+</style>
+
+<div id='map'></div>
+<div class='button-container'>
+    <button class='round-top button icon caret-up' id='up'></button>
+    <button class='round-left button icon caret-left' id='left'></button>
+    <button class='unround button icon caret-down' id='down'></button>
+    <button class='round-right button icon caret-right' id='right'></button>
+</div>
+
+<script>
+    //Initialize the map
+    var map = new mapboxgl.Map({
+        container: 'map',
+        style: 'mapbox://styles/mapbox/dark-v9',
+        center: [-87.6298, 41.8781],
+        zoom: 17,
+        bearing: -12,
+        pitch: 60
+    });
+
+    var deltaDistance = 100; // how far the map moves in pixels when the up or down arrow is clicked
+    var deltaDegrees = 25; // rotation of the map in degrees when the left or right arrow is clicked
+
+    // prevent zooming and panning that is not done
+    // with the keyboard arrows or buttons
+    map.boxZoom.disable();
+    map.dragPan.disable();
+    map.doubleClickZoom.disable();
+    map.scrollZoom.disable();
+    map.keyboard.disable();
+    map.touchZoomRotate.disable();
+
+    function easeTo(t) {
+        return t * (2 - t);
+    }
+
+    function move(pos, bearing) {
+        if (bearing) {
+            map.easeTo({
+                bearing: pos,
+                easing: easeTo
+            });
+        } else {
+            map.panBy(pos, {
+                easing: easeTo
+            });
+        }
+    }
```

Comment:
This `move` function does not provide any abstraction over the `Map#easeTo` method. Can we eliminate this function in favor of `Map#easeTo`? Note that `Map#panBy` calls `Map#easeTo` internally.